Apps/frontend/src/shader/indicatorShader.js

```diff
@@ -0,0 +1,31 @@
+export const fragmentShader = `#version 300 es
+precision mediump float;
+
+in vec4 v_colour;
```

Comment:
I guess `o_colour` is for out colour, what is `v_colour` short for?

Reply:
As far as I understood, v_colour is the color set by the vertex shader and then passed to the fragment shader.
So the vertex shader sets the initial color and the fragment shader returns a color which is dependent of the position computed by the vertex shader.
As there is no perspective or lighting in out 2D view, the fragment shader just returns the color it was given.
So maybe the "v" could stand for "vertex". (Not sure, still a noob in WebGL...)